build.sh

```diff
@@ -11,12 +11,12 @@ function copySrcToBuild {
   rm -rf build/
   mkdir build/
 
-  cp -r src/ build/
+  cp -r src/. build/
```

Comment:
I got this from a SO question, and it worked. But yeah, my bash foo is very weak. I'll just move these script changes to another PR, so we can merge the rest safely.